crates/zed/src/open_listener.rs

```diff
@@ -131,7 +129,7 @@ impl OpenListener {
             .flat_map(|url| url.strip_prefix("file://"))
             .map(|url| {
                 let decoded = urlencoding::decode_binary(url.as_bytes());
-                PathBuf::from(OsStr::from_bytes(decoded.as_ref()))
+                PathBuf::try_from_bytes(decoded.as_ref()).unwrap()
```

Comment:
Note: `try_from_bytes` always returns `Ok` on `[cfg(unix)]`.

Reply:
Because we don't control the input to this method, we should not panic if the bytes are invalid, but just ignore them. I think the following would be better:

```suggestion
          .flat_map(|url| {
                let decoded = urlencoding::decode_binary(url.as_bytes());
                PathBuf::try_from_bytes(decoded.as_ref()).log_err()
           })
```

Reply:
I fixed this